frontend-admin/src/pages/vans/vans-page.tsx

```diff
@@ -1,15 +1,34 @@
 import React from 'react';
+import { useQuery } from 'react-query';
 import Card from '../../components/card/card';
 import './vans-page.scss';
 
+interface Van {
+  vanId: number;
+  routeId: number;
+  wheelchair: boolean;
+}
+
+const fetchVans = async () => {
+  const response = await fetch('http://localhost:8000/vans');
+  const data = await response.json();
+  const van_data = data["vans"] as Van[];
```

Comment:
Will this throw an error if it's not in the right structure?

Reply:
unsure, just getting stuff done for the demo